### PR TITLE
CTCTRADERS-445: Removed UI

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: conf/logback.xml [hunk 1]
```diff
@@ -1,59 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
-    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <file>logs/common-transit-convention-trader-at-destination-stub.log</file>
```
Comment (ashwinirudrappa) [inline conf/logback.xml:5]: @MatthewAlexanderBeck Do we not need FileAppender

File: conf/logback.xml [hunk 2]
```diff
@@ -1,59 +1,13 @@
...
-
-    <logger name="application" level="DEBUG"/>
-
-    <logger name="connector" level="TRACE">
-        <appender-ref ref="STDOUT"/>
-    </logger>
-
-    <root level="INFO">
+    <root level="OFF">
         <appender-ref ref="FILE"/>
```
Comment (ashwinirudrappa) [inline conf/logback.xml:11]: @MatthewAlexanderBeck as `<appender name="FILE" class="ch.qos.logback.core.FileAppender">` is removed what  <appender-ref ref="FILE"/>
```suggestion
        <appender-ref ref="FILE"/>
```
will refer

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -6,22 +6,15 @@ import sbt._
...
-    "uk.gov.hmrc"       %% "logback-json-logger"           % "4.6.0",
-    "uk.gov.hmrc"       %% "play-health"                   % "3.14.0-play-26",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping" % "1.2.0-play-26",
-    "uk.gov.hmrc"       %% "bootstrap-play-26"             % "1.7.0",
-    "uk.gov.hmrc"       %% "play-whitelist-filter"         % "3.1.0-play-26",
-    "uk.gov.hmrc"       %% "play-nunjucks"                 % "0.23.0-play-26",
-    "uk.gov.hmrc"       %% "play-nunjucks-viewmodel"       % "0.8.0-play-26",
-    "org.webjars.npm"   % "govuk-frontend"                 % "3.3.0"
+    "uk.gov.hmrc"       %% "bootstrap-play-26"             % "1.4.0",
+    "uk.gov.hmrc"       %% "bootstrap-play-26"             % "1.7.0"
```
Comment (ashwinirudrappa) [inline project/AppDependencies.scala:10]: @MatthewAlexanderBeck Why we want different version of bootstraps

Comment (MatthewAlexanderBeck) [reply project/AppDependencies.scala:10]: Looks like a typo, amended it now

**Pull request merged**